pkg/skaffold/update/update_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/testutil"
+	"github.com/blang/semver"
+)
+
+func TestIsUpdateCheckEnabled(t *testing.T) {
+	var tests = []struct {
+		description       string
+		updateCheckEnvVar string
+		expected          bool
+	}{
+		{
+			description:       "env is empty",
+			updateCheckEnvVar: "",
+			expected:          true,
+		},
+		{
+			description:       "env is true",
+			updateCheckEnvVar: "true",
+			expected:          true,
+		},
+		{
+			description:       "env is false",
+			updateCheckEnvVar: "false",
+			expected:          false,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			reset := testutil.SetEnvs(t, map[string]string{
+				"SKAFFOLD_UPDATE_CHECK": test.updateCheckEnvVar,
+			})
+			defer reset(t)
+
+			testutil.CheckDeepEqual(t, test.expected, IsUpdateCheckEnabled())
+		})
+	}
+}
+
+func TestGetLatestAndCurrentVersion(t *testing.T) {
```

Comment:
I could not add a case when `GetLatestAndCurrentVersion` succeeds since `version` variable in version package is enexported and `version.ParseVersion` for current version always fails.